hvantk/algorithms/qtlcascade/gwas_coloc.py

```
@@ -0,0 +1,278 @@
+    """
+    gwas_min_p = min((v[2] for v in gwas.values()), default=float("nan"))
+    rows = []
+    for ensg, recs in eqtl.items():
+        b1, s1, b2, s2, eqtl_ps = [], [], [], [], []
+        for (pos, eref, ealt), ebeta, ese, epval in recs:
+            # Match order-independently: sources may store ref/alt swapped.
+            # The ABF kernel uses z^2 (sign-independent), so no beta flip needed.
+            g = gwas.get((pos, eref, ealt)) or gwas.get((pos, ealt, eref))
+            if g is None:
+                continue
+            b1.append(g[0]); s1.append(g[1])
```

Ruff (0.15.15)

[error] Multiple statements on one line (semicolon)

(E702)